packages/web/src/components/organisms/AuthenticateForm/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect, useCallback } from 'react'
+import Link from 'next/link'
+import { Select } from 'antd'
+import { Form, Input, Button } from 'antd'
+import { useCreateProperty } from 'src/fixtures/dev-kit/hooks'
+
+interface Props {
+  walletAddress: string
+}
+
+const { Option } = Select
+
+const PropertyCreateForm = ({ walletAddress }: Props) => {
+  const [form] = Form.useForm()
+  useEffect(() => {
+    form.setFieldsValue({
+      author: walletAddress
+    })
+  }, [form, walletAddress])
+  const { createProperty } = useCreateProperty()
+  const handleCreateProperty = useCallback(
+    (name: string, symbol: string, author: string) => createProperty(name, symbol, author),
+    [createProperty]
+  )
+
+  const onFinishFailed = (errorInfo: any) => {
+    console.log('Failed:', errorInfo)
+  }
+
+  return (
+    <div style={{ marginTop: '18px' }}>
+      <Form
+        labelCol={{ span: 6 }}
+        wrapperCol={{ span: 18 }}
+        form={form}
+        name="basic"
+        initialValues={{ remember: true }}
+        onFinish={({ name, symbol, author }) => {
+          handleCreateProperty(name, symbol, author)
+        }}
+        onFinishFailed={onFinishFailed}
+      >
+        <Form.Item label="Name" name="name" rules={[{ required: true, message: 'Please input name.' }]}>
+          <Input placeholder="Name" />
+        </Form.Item>
+        <Form.Item label="Symbol" name="symbol" rules={[{ required: true, message: 'Please input symbol.' }]}>
+          <Input placeholder="Symbol" />
+        </Form.Item>
+        <Form.Item label="Author" name="author">
+          <Input type="text"></Input>
+        </Form.Item>
+        <Form.Item>
+          <Button type="primary" htmlType="submit">
+            Submit
+          </Button>
+        </Form.Item>
+      </Form>
+    </div>
+  )
+}
+
+export const AuthenticateForm = ({ market }: { market: string }) => {
+  const [isOpen, setIsOpen] = useState(false)
+  const walletAddress = 'wallet-address'
+  const [property, setProperty] = useState<string>()
+
+  const onChange = (value: string) => setProperty(value)
+
+  const onSearch = (val: string) => console.log('search:', val)
```

Comment:
Work In Progress. If we need to search word on select box, I'll implement this function.

Reply:
I understood that plan.